.github/workflows/ci-standard-checks.yml

```diff
@@ -4,23 +4,27 @@ on:
     branches:
       - main
   pull_request:
-    types: [opened, edited, synchronize, reopened]
+    types:
+      - opened
+      - edited
+      - synchronize
+      - reopened
+      - ready_for_review
     branches:
       - main
-
 jobs:
   ci-standard-checks:
-    runs-on: [ubuntu-latest]
-
+    runs-on:
+      - self-hosted
+      - automated-checks-ephemeral
     steps:
       - name: Check Out Source Code
         uses: actions/checkout@v2
         with:
           fetch-depth: 0
-
       - name: CI Standard Checks
         uses: Typeform/ci-standard-checks@v1
         with:
           githubToken: ${{ secrets.GITHUB_TOKEN }}
-          dockerUsername: ${{ secrets.GITLEAKS_DOCKER_USERNAME }}
-          dockerPassword: ${{ secrets.GITLEAKS_DOCKER_PASSWORD }}
+          dockerUsername: ${{ secrets.DOCKER_USERNAME }}
+          dockerPassword: ${{ secrets.DOCKER_PASSWORD }}
```

Comment:
on webflow we used these secrets

Reply:
You can refer to this as the source of truth, so probably these `docker` variables are no longer necessary.

https://github.com/Typeform/.github/blob/main/workflow-templates/ci-standard-checks.yml